README.md

```diff
@@ -51,12 +53,23 @@ Alternatively, you can pre-compile the eBPF program, and provide it to the `trac
+Once you have the eBPF program artifact, you can provide it to Tracee in any of the locations mentioned above. In this case, the full Docker image can be replaced by the lighter-weight `aquasec/tracee:slim` image. This image cannot build the eBPF program on its own, and is meant to be used when you have already compiled the eBPF program beforehand.
+
+#### Running in container
+
+Tracee uses a filesystem directory, by default `/tmp/tracee` to capture runtime artifacts, internal components, and other miscellaneous. When running in a container, it's useful to mount this directory in, so that the artifacts are accessible after the container exists. For example, you can add this to the docker run command `-v /tmp/tracee:/tmp/tracee`.
+
+If running in a container, regardless if it's the full or slim image, it's advisable to reuse the eBPF program across runs by mounting it from the host to the container. This way if the container builds the eBPF program it will be persisted on the host, and if the eBPF program already exists on the host, the container will automatically discover it. If you've already mounted the `/tmp/tracee` directory from the host, you're good to go, since Tracee by default will use this location for the eBPF program. You can also mount the eBPF program file individually if it's stored elsewhere (e.g in a shared volume), for example: `-v /path/to/tracee.ebpf.1_2_3.4_5_6.o:/some/path/tracee.ebpf.1_2_3.4_5_6.o -e TRACEE_BPF_FILE=/some/path`. 
+
+When using the `--capture exec` option, Tracee needs access to the host PID namespace. For Docker, add `--pid=host` to the run command.
+
+If you are building the eBPF program in a container, you'll need to make the kernel headers available in the container. The quickstart example has wide mounts that works in a variety of cases, for demonstration purposes. If you want, you can narrow those mounts down to a directory that contains the headers on your setup, for example: `-v /path/to/headers:/myheaders -e KERN_SRC=/myheaders`. As mentioned before, a better practice for production is to pre-compile the eBPF program, in which case the kernel headers are not needed at runtime.
 
-### Permissions
+#### Permissions
 
-If you use the Tracee binary, you'll need to run it with root permissions in order to load the eBPF code. 
-If you use the Docker container, you should run it with the `--privileged` flag.
+Tracee needs to run with sufficient privileges to load and attach the eBPF program. This means running as `root`, or with the `--privileged` flag if running with Docker.  
+For better control, you can run with the specific capabilities `SYS_ADMIN` and `CAP_SYS_RESOURCE`.  
```

Comment:
Starting from kernel 5.8, there are new CAP_BPF+CAP_TRACING which should be used instead of SYS_ADMIN
CAP_SYS_RESOURCE is used to change the ulimit in order to be able to make bpf maps